ConsoleApp1/list.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ConsoleApp1
+{
+    class list
```

Comment:
All class names in .net must be capitalized.
This is not a List of figures. There is no collection in it